testsuite/forge-cli/src/main.rs

```diff
@@ -608,23 +663,18 @@ fn run_consensus_only_three_region_simulation() -> ForgeConfig {
             helm_values["chain"]["epoch_duration_secs"] = (24 * 3600).into();
         }))
         .with_node_helm_config_fn(Arc::new(|helm_values| {
```

Comment:
should we provide similar function to what is with_init_config in Smoke tests, to make this less verbose?

i.e. this gets changed with:

```
 .with_init_config(Arc::new(|conf| {
            update_mempool_config_practically_non_expiring(conf.mempool);
            conf.consensus.max_sending_block_txns = 5000;
            conf.consensus.max_receiving_block_txns = 30000;
            conf.consensus.max_sending_block_bytes = 3 * 1024 * 1024;
            update_state_sync_config_execute_transactions(conf.state_sync);
        }))
```

and with_init_config then first creates default NodeConfig, passes it here, and then calls 
helm_values["validator"]["config"] = override_yaml_from_node_config(config);

This removes also the ..Default::default() bit , which is annoying.

Reply:
I agree this is less verbose. But it does completely remove the possibility of hard-coding already default values. I suppose we could include both options, but then it gets a bit confusing (it matters which function is called first)

Reply:
If really needed, we can have function provide two arguments: (config, base_config), and most uses will have _ for the second one.

Reply:
Hmm, as it is now, we'd need two versions:
with_init_validator_config
with_init_fullnode_config
and also some tests change other helm_values, e.g., helm_values["chain"] so we'd still need a way to do that.

Reply:
It would be good to have Chain as a struct as well (so that fields are typed and stuff).
and having 3 functions, for each of the validator, fullnode and chain configs - sounds good?

Reply:
Oh nevermind, chain is part of genesis. The only other node helm values that are touched are in `multiregion_benchmark_test`. I think I'll just separate those into its own part of the builder and we could generalize them if people actually need them later.